scripts/convert.js

```diff
@@ -5,10 +5,7 @@
 
 import csv from 'csv';
 import getStdin from 'get-stdin';
-import {
-  INDICATORS,
-  CITIES,
-} from '../src/constants';
+import INDICATORS from '../src/constants/indicators';
```

Comment:
This needs to import `INDICATORS` directly from the `indicators.js` file, since the `cities.js` file now includes JSX which is invalid JavaScript (so running this convert script would break).

Reply:
Seems to me that the moment at which one would be alerted to invalid data is with an error during data import - not an warning during app runtime. Plus I have reservations about use of JSX (see my comments elsewhere in this PR).

Reply:
Agreed, let's sort out the JSX vs markdown thing first then look at this.